mx.py

```diff
@@ -9341,6 +9341,21 @@ def log(msg=None):
     else:
         print msg
 
+def log_error(msg=None):
+    """
+    Write an error message to the console.
+    All script output goes through this method thus allowing a subclass
+    to redirect it.
+    """
+    if msg is None:
+        sys.stderr.write(os.linesep)
+    elif sys.platform.startswith('linux') or\
+         sys.platform in ['darwin', 'freebsd']:
+        # On unix systems, we can use ANSI escape sequences
+        sys.stderr.write('\033[91m' + msg.encode('utf8') + '\033[0m' + os.linesep)
```

Comment:
How does this look in captured log files? Maybe we should only colorize if in an interactive console.